lib/webmock/http_lib_adapters/net_http.rb

```diff
@@ -285,7 +285,7 @@ def rbuf_fill
         end
 
         super
-
+      ensure
```

Comment:
In Ruby 2.6, the `TracePoint#enable` takes `target:` keyword argument.
We can use `TracePoint#enable` with the `target:` to enable the TracePoint for the specific method.
What do you think?
```rb
        trace.enable(target: WebMock::HttpLibAdapters::NetHttpAdapter::OriginalNetBufferedIO.instance_method(:rbuf_fill)) do
          super
        end
```

Reply:
you == me?

It is better. But I think this TracePoint approach is danger, even with this limitation.